src/Result.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+
+namespace PivotLibre\Tideman;
+
+/**
+ * Class Result
+ * Instances of this class describe the final result of the election (@see Result::getRanking()) and a few other
+ * intermediate results for reporting and visualization purposes.
+ *
+ * @package PivotLibre\Tideman
+ */
+class Result
+{
+    protected $ranking;
```

Comment:
I don't know how PHP types work.  Can you put them here, as with the getters/setters?

Reply:
Unfortunately, no.